scan-bar.go

```diff
@@ -43,6 +43,9 @@ func fixateScanBar(text string, width int) string {
 // Progress bar function report objects being scaned.
 type scanBarFunc func(string)
 
+func discardScanBarFactory(s string) {
```

Comment:
Yes, the discardScanBarFactory is a function that won't show the output, so we don't have to write all ifs and elses.